bin/inx-chronicle/src/main.rs

```diff
@@ -135,6 +136,15 @@ impl HandleEvent<Report<InxListener>> for Launcher {
                                 .await;
                         }
                     },
+                    InxListenerError::InxConnection(e) => match e {
+                        InxError::TransportFailed => {
+                            let wait_interval = self.inx_connection_retry_interval;
+                            log::info!("Retrying INX connection in {} seconds.", wait_interval.as_secs_f32());
+                            tokio::time::sleep(wait_interval).await;
```

Comment:
Instead of blocking the event loop here, this should probably spawn a task which will send the message to the Launcher handle after sleeping.